src/wp-includes/formatting.php

```diff
@@ -5771,7 +5771,7 @@ function _print_emoji_detection_script() {
 		 *
 		 * @param string $url The emoji base URL for svg images.
 		 */
-		'svgUrl'  => apply_filters( 'emoji_svg_url', 'https://s.w.org/images/core/emoji/13.1.0/svg/' ),
+		'svgUrl'  => apply_filters( 'emoji_svg_url', 'https://s.w.org/images/core/emoji/14.0.1/svg/' ),
```

Comment:
In `tests/phpunit/tests/formatting/emoji.php`, line 9 and 10 should also be updated to change `13.1.0` to `14.0.1`.

Reply:
Ah, thank you. Updated in 3e9014396a